x-pack/test/functional/apps/ml/short_tests/notifications/notification_list.ts

```diff
@@ -92,5 +96,22 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await browser.refresh();
       await ml.notifications.assertNotificationErrorsCount(0);
     });
+
+    it('support custom sorting for notifications level', async () => {
+      await ml.navigation.navigateToNotifications();
+      await ml.notifications.table.waitForTableToLoad();
+
+      await PageObjects.timePicker.pauseAutoRefresh();
+      const fromTime = moment().subtract(1, 'week').format(timepickerFormat);
+      const toTime = moment().format(timepickerFormat);
+      await PageObjects.timePicker.setAbsoluteRange(fromTime, toTime);
+
+      await ml.notifications.table.waitForTableToLoad();
+
+      await ml.notifications.table.sortByField('level', 1, 'desc');
+
+      const rows = await ml.notifications.table.parseTable();
+      expect(rows[0].level).to.eql('error');
```

Comment:
Suggestion to add another step here to sort by 'level' ascending and check if it's 'info' if possible.

Reply:
Added in https://github.com/elastic/kibana/pull/153462/commits/6b6f375965b3047a6a6e30f20287736932a2e084